Mathlib/CategoryTheory/Limits/Shapes/Pullback/CommSq.lean

```diff
@@ -249,6 +264,38 @@ theorem of_hasBinaryProduct [HasBinaryProduct X Y] [HasZeroObject C] [HasZeroMor
   convert @of_is_product _ _ X Y 0 _ (limit.isLimit _) HasZeroObject.zeroIsTerminal
     <;> subsingleton
 
+section
+
+variable {P': C} {fst' : P' ⟶ X} {snd' : P' ⟶ Y}
+
+/-- Any object at the top left of a pullback square is isomorphic to the object at the top left
+of any other pullback square with the same cospan. -/
+noncomputable def isoIsPullback (h : IsPullback fst snd f g) (h' : IsPullback fst' snd' f g) :
+    P ≅ P' :=
+  IsLimit.conePointUniqueUpToIso h.isLimit h'.isLimit
+
+@[simp]
```

Comment:
This also applies to other lemmas (basically anytime the LHS is a composition):
```suggestion
@[reassoc (attr := simp)]
```

Reply:
Thanks, forgot about that!